src/models/network.ts

```diff
@@ -39,8 +40,9 @@ const mainnet = Networks.add({
   pubkeyhash: versionBytes['mainnet']['p2pkh'],
   privatekey: 0x80,
   scripthash: versionBytes['mainnet']['p2sh'],
+  bech32prefix: 'ht',
   xpubkey: 0x0488b21e,
-  xprivkey: 0x0488ade4,
+  xprivkey: 0x0488ade5,
```

Comment:
What is this parameter for? And why are we just increasing it by one? Should we create a value for Hathor? How? Randomly?

Reply:
This parameter only exists to signal which network the serialized xprivkey belongs to. The onty constraint i could find is that it must be 4 bytes long and must not be the same as the xpubkey (from the network object). This will **NOT** change the generated addresses, it only affects the metadata used internally by bitcore.